### PR TITLE
Implement stddev_samp and stddev_pop ppl stats function

### PR DESCRIPTION
**Pull request by salyh**
### Description
Implement stddev_samp and stddev_pop ppl stats function

### Issues Resolved
https://github.com/opensearch-project/opensearch-spark/issues/483

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: ppl-spark-integration/src/main/java/org/opensearch/sql/ppl/utils/AggregatorTranslator.java [hunk 1]
```diff
@@ -35,6 +35,10 @@ static Expression aggregator(org.opensearch.sql.ast.expression.AggregateFunction
                 return new UnresolvedFunction(seq("COUNT"), seq(arg),false, empty(),false);
             case SUM:
                 return new UnresolvedFunction(seq("SUM"), seq(arg),false, empty(),false);
+            case STDDEV_POP:
```
Comment (YANG-DB) [inline ppl-spark-integration/src/main/java/org/opensearch/sql/ppl/utils/AggregatorTranslator.java:38]: @salyh please add logical plan tests and full query IT tests

File: ppl-spark-integration/src/main/java/org/opensearch/sql/ppl/utils/AggregatorTranslator.java [hunk 2]
```diff
@@ -35,6 +35,10 @@ static Expression aggregator(org.opensearch.sql.ast.expression.AggregateFunction
...
+                return new UnresolvedFunction(seq("STDDEV_POP"), seq(arg), aggregateFunction.getDistinct(), empty(),false);
+            case STDDEV_SAMP:
```
Comment (YANG-DB) [inline ppl-spark-integration/src/main/java/org/opensearch/sql/ppl/utils/AggregatorTranslator.java:40]: @salyh please add logical plan tests and full query IT tests

**Review by YANG-DB — CHANGES_REQUESTED** [review]

File: ppl-spark-integration/src/main/java/org/opensearch/sql/ppl/utils/AggregatorTranslator.java (hunk 2 above)
Comment (kt-eliatra) [reply ppl-spark-integration/src/main/java/org/opensearch/sql/ppl/utils/AggregatorTranslator.java:40]: done

**Review by YANG-DB — COMMENTED** [review]
looks good
please also add the command description in the readme section at the last part of the [page](https://github.com/opensearch-project/opensearch-spark/blob/main/ppl-spark-integration/README.md)

**Comment by YANG-DB** [comment]
hi
plz also run ` sbt scalafmtAll  ` for solving the [failed CI / build](https://github.com/opensearch-project/opensearch-spark/actions/runs/10384781425/job/28820984141?pr=549#step:5:28)

**Comment by YANG-DB** [comment]
hi
plz also run ` sbt scalafmtAll  ` for solving the [failed CI / build](https://github.com/opensearch-project/opensearch-spark/actions/runs/10384781425/job/28820984141?pr=549#step:5:28)

**Review by YANG-DB — APPROVED** [review]

**Pull request merged**